iceoryx_hoofs/test/moduletests/test_vocabulary_string.cpp

```diff
@@ -2939,48 +2706,6 @@ TEST(String100, FindFirstOfForNotIncludedStringLiteralFails)
     EXPECT_THAT(res.has_value(), Eq(false));
 }
 
-TEST(String100, FindFirstOfForSTDStringInNotEmptyStringWorks)
```

Comment:
@FerdinandSpitzschnueffler not for this PR but do we need the `String100` test and shouldn't these test cases already be covered by typed tests?

Reply:
The initial reason for the `String100` tests was that some tests require a `sut` capacity > 1, e.g. `UnsafeCharToStringConstrIncludingNullCharWithCountResultsInSizeCount`. We could rewrite the tests to typed tests but 3 years ago I thought the `String100` approach was a good idea...
To answer your question: I doubt that the `String100` tests are already covered by typed tests.

Reply:
Okay, then we can keep it but maybe rename them at some time. They remind me of `CString100` 😱

Reply:
Something for a follow-up PR. Closing.